### ISSUE TITLE
[4.x]: `perItemDiscount` weird behaviour

### ISSUE THREAD

**Issue by vandres** [craftcms/commerce#2937]
### Description

I think this might be related to i18n. My user is set to German, the base currency is EUR.

When working with Money fields, usually we see a "," as decimal separator in the CP. The `perItemDiscount` though works weirdly for us. When I enter a number with ",", it works fine. Lets say "12,5" But when I reopen the discount rule, the field says "12.5". Which is theroretically the thousand separator. When I resave the field, the field says now "125", which is also used as a value. We tested only this field, maybe others are also affected.

In the same dialog "CMD/Ctrl + S" ist not working. After saving like this, the rule is left back to the overview.

### Steps to reproduce

1. Create a discount rule

### Expected behavior

- The discount value should be loaded with "," in our case 
- "CMD/Ctrl + S" should work

### Actual behavior

- The discount value is loaded with "."
- "CMD/Ctrl + S" is not working as expected

### Craft CMS version

4.2.1.1

### Craft Commerce version

4.1.0

### PHP version

8.1

### Operating system and version

MacOS, Debian Docker

### Database type and version

MariaDB 10.5

### Image driver and version

-

### Installed plugins and versions

```json
  "require": {
    "craftcms/cms": "^4.2.0.2",
    "craftcms/commerce": "^4.1",
    "craftcms/redactor": "^3.0",
    "firephp/firephp-core": "^0.5.3",
    "mmikkel/cp-field-inspect": "^1.4",
    "nystudio107/craft-vite": "^4.0",
    "php-http/curl-client": "^2.2",
    "spicyweb/craft-neo": "^3.2",
    "typesense/typesense-php": "^4.8",
    "vaersaagod/geomate": "^2.0",
    "verbb/expanded-singles": "^2.0",
    "verbb/field-manager": "^3.0",
    "verbb/tablemaker": "^4.0",
    "verbb/wishlist": "^2.0",
    "vlucas/phpdotenv": "^5.4.0"
  },
  "require-dev": {
    "roave/security-advisories": "dev-master",
    "deployer/deployer": "^7.0",
    "yiisoft/yii2-shell": "^2.0.3"
  },
```

**Comment by pdaleramirez** [comment]
Thank you for reporting this issue. We'll be working on a fix for this.

**Comment by pdaleramirez** [comment]
@vandres Fixed now on Commerce v4.1.2

**Comment by vandres** [comment]
@pdaleramirez  Thank you! Can confirm that's working now :)
